flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/gateway/context/SessionContextTest.java

```diff
@@ -53,31 +54,31 @@
 import static org.assertj.core.api.HamcrestCondition.matching;
 
 /** Test {@link SessionContext}. */
-public class SessionContextTest {
+class SessionContextTest {
 
-    @ClassRule public static TemporaryFolder tempFolder = new TemporaryFolder();
+    @TempDir private static Path tempFolder;
 
     private static File udfJar;
 
     private SessionContext sessionContext;
 
-    @BeforeClass
+    @BeforeAll
     public static void prepare() throws Exception {
         udfJar =
                 UserClassLoaderJarTestUtils.createJarFile(
-                        tempFolder.newFolder("test-jar"),
+                        Files.createTempDirectory(tempFolder, "test-jar").toFile(),
                         "test-classloader-udf.jar",
                         GENERATED_LOWER_UDF_CLASS,
                         String.format(GENERATED_LOWER_UDF_CODE, GENERATED_LOWER_UDF_CLASS));
     }
 
-    @Before
+    @BeforeEach
     public void setup() {
```

Comment:
```suggestion
    void setup() {
```
could be package private